prijateli_tree/app/templates/student_info.html

```diff
@@ -0,0 +1,4 @@
+<div class="container-sm general-container">
+  <p class="text-end">{{ player_name }}</p>
```

Comment:
The header should also include round information.  You can use columns to separate the text. <p> tags are creating too much space between the words. Is there another way?